src/backend/app.js

```diff
@@ -19,12 +21,89 @@ app.use(express.json());
 
 app.use(cors());
 
+router.use("/reservations", reservationsRouter);
 router.use("/meals", mealsRouter);
 
+//Respond with all meals in the future (relative to the when datetime) - /future-meals
+
+app.get("/future-meals", async (req, res) => {
```

Comment:
I would say that those endpoints should be a part of `meals` module and the URLs should look like this `meals/future`, `meals/past` etc :)  it seems logical, cause they refer to the same collection of meals, they are just bit more specific.